<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateEthMultisigContractDialog</class>
 <widget class="QDialog" name="CreateEthMultisigContractDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="numLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>/ 0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>500</width>
     <height>240</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>已添加地址</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="createBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>创建</string>
   </property>
  </widget>
  <widget class="QToolButton" name="addAddressBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>添加ETH地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>需要签名数量</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
